lib/zooniverse_data/projects/wise.rb

```diff
@@ -0,0 +1,21 @@
+module ZooniverseDaya
+  module Projects
+    class Wise
+      include Helpers
+
+      def customize_subject
+        new_locations = {}
+
+        entry.location['standard'].each do |path|
+          image_type = path.match(/.*_(?<image_type>[a-zA-Z0-9]+)\.png$/)
+          image_type = image_type[:image_type]
+          new_locations[image_type] = path
+        end
+
+        new_locations['standard'] = new_locations['wise4']
```

Comment:
It looks like this will duplicate the location (i.e. wise4 will duplicate standard).  Is that okay or did you mean


``` ruby
new_locations['standard'] = new_locations.delete 'wise4'
```